src/main/java/com/baidu/hugegraph/backend/store/tikv/TikvStdSessions.java

```diff
@@ -355,6 +375,15 @@ public void put(String table, byte[] key, byte[] value) {
             this.putBatch.put(this.key(table, key), ByteString.copyFrom(value));
         }
 
+        @Override
+        public void put(String table, byte[] key, byte[] value, long ttl) {
+            Map<ByteString, ByteString> batch =
+                            this.putBatchTTL.getOrDefault(ttl,
+                                 new HashMap<ByteString, ByteString>());
```

Comment:
call putBatchTTL.get() and don't need to create map if putBatchTTL.get() return non-null result